core/tests/protractor_desktop/profileFeatures.js

```diff
@@ -0,0 +1,17 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
```

Comment:
Ah, shouldn't this be removed?

Reply:
I just added blank files for all the e2e suites that we should ideally add. The idea being, in the follow up issue I am planning to file, I wanted to write: Test for <functionality> goes in <filename>. I hope it is ok to have blank test files which will be populated soon.

Reply:
I do not have any issues with the blank test file being here but we should not add this to the travis suite yet since even though it is blank, it takes 3 minutes for the setup etc. [here](https://travis-ci.org/oppia/oppia/jobs/484427119). Thanks!

Reply:
Ah I see, that again, I preemptively added, to have the e2e starter project just happen, without the need to change travis.yml. If you want me to remove the travis suites corresponding to these files, I can do that (or possibly disable them for now). Would that be better?

Reply:
Yes, I think removing these suites from travis altogether atm sounds good! Thanks :)